src/simulation/clues/Elite.ts

```diff
@@ -114,7 +114,7 @@ export const EliteRareTable = new LootTable()
 	.add('Giant boot')
 	.add("Rangers' tunic")
 	.add('Monocle')
-	.add(EliteMegaRareTable)
+	.add(EliteMegaRareTable, 1, 2)
```

Comment:
.add(Loottable/itemname, amount, weight)

So if you want to 2 rolls.
 .add(EliteMegaRareTable, 2, 1)

Reply:
Rephrased the description in "Changes" to better describe what was changed.